jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -271,14 +273,28 @@ private void extractTar() {
   }
 
   private void assignLocalImageResult(Future<LocalImage> localImage) {
-    results.baseImageLayers = executorService.submit(() -> localImage.get().layers);
+
+    Future<List<Future<PreparedLayer>>> localBaseImageLayers =
+        executorService.submit(() -> localImage.get().layers);
+
     results.baseImageAndRegistryClient =
         executorService.submit(
-            () ->
-                LocalBaseImageSteps.returnImageAndRegistryClientStep(
-                        realizeFutures(results.baseImageLayers.get()),
-                        localImage.get().configurationTemplate)
-                    .call());
+            () -> {
```

Comment:
And probably you don't need `{ return ... }` but keep the original syntactic sugar `() -> expr`.

Reply:
Adding back the ` sugar ` so that our code can be ` sweet ` again hahah . Done.